src/lib/types/Events.ts

```diff
@@ -597,6 +628,9 @@ declare module 'discord.js' {
 
 		[SapphireEvents.PluginLoaded]: [hook: PluginHook, name: string | undefined];
 
+		[SapphireEvents.InteractionHandlerParseSuccess]: [option: Option.None | Option.Some<unknown>, payload: InteractionHandlerParseSuccess];
```

Comment:
Can be simplified to this:
```suggestion
		[SapphireEvents.InteractionHandlerParseSuccess]: [option: Option<unknown>, payload: InteractionHandlerParseSuccess];
```